Cargo.toml

```diff
@@ -38,7 +38,7 @@ pyo3 = { version = "0.20", features = ["auto-initialize"]}
 rand = "0.8"
 redis = { version = "0.24", default-features = false, features = ["acl", "tokio-rustls-comp", "cluster", "cluster-async", "connection-manager", "keep-alive", "script"]}
 regex = "1.0"
-reqwest = { version = "0.11", features = ["gzip", "cookies"] }
+reqwest = { version = "0.11", default-features = false, features = ["rustls-tls", "gzip", "cookies"] }
```

Comment:
This removes the OpenSSL dependency?
I presume this doesn't do anything for the OpenSSL license issue we discussed because rustls still depends on ring?

Reply:
Yes and yes :+1: